doc/pset.mediawiki

```diff
@@ -0,0 +1,803 @@
+*** <tt>{porCommitment}</tt>
+
+* Type: Proprietary Use Type <tt>PSBT_IN_PROPRIETARY = 0xFC</tt>
+** Key: Variable length identifier prefix, followed by a subtype, followed by the key data itself.
+*** <tt>{0xFC}|<prefix>|{subtype}|{key data}</tt>
+** Value: Any value data as defined by the proprietary type user.
+*** <tt><data></tt>
+
+The currently defined elements per-input proprietary types are as folows:
+
+* Type: Issuance Value <tt>PSET_IN_ISSUANCE_VALUE = 0x00</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x00}</tt>
+** Value: The explicit little endian 64-bit integer for the value of this issuance. This is mutually exclusive with <tt>PSET_IN_ISSUANCE_VALUE_COMMITMENT</tt>
+*** <tt>{value}</tt>
+
+* Type: Issuance Value Commitment <tt>PSET_IN_ISSUANCE_VALUE_COMMITMENT = 0x01</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x01}</tt>
+** Value: The 33 byte Value Commitment. This is mutually exclusive with <tt>PSBT_IN_ISSUANCE_VALUE</tt>.
+*** <tt>{value commitment}</tt>
+
+* Type: Issuance Value Rangeproof <tt>PSET_IN_ISSUANCE_VALUE_RANGEPROOF = 0x02</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x02}</tt>
+** Value: The rangeproof
+*** <tt>{rangeproof}</tt>
+
+* Type: Issuance Inflation Keys Rangeproof <tt>PSET_IN_ISSUANCE_KEYS_RANGEPROOF = 0x03</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x03}</tt>
+** Value: The rangeproof
+*** <tt>{rangeproof}</tt>
+
+* Type: Peg-in Transaction <tt>PSET_IN_PEG_IN_TX = 0x04</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x04}</tt>
+** Value: The Peg-in Transaction serialized without witnesses.
+*** <tt>{tx}</tt>
+
+* Type: Peg-in Transaction Output Proof <tt>PSET_IN_PEG_IN_TXOUT_PROOF = 0x05</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x05}</tt>
+** Value: The transaction output proof for the Peg-in Transaction.
+*** <tt>{txoutproof}</tt>
+
+* Type: Peg-in Genesis Hash <tt>PSET_IN_PEG_IN_GENESIS = 0x06</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x06}</tt>
+** Value: The 32 byte genesis hash for the Peg-in Transaction.
+*** <tt>{hash}</tt>
+
+* Type: Peg-in Claim Script <tt>PSET_IN_PEG_IN_CLAIM_SCRIPT = 0x07</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x07}</tt>
+** Value: The claim script for the Peg-in Transaction.
+*** <tt>{script}</tt>
+
+* Type: Peg-in Value <tt>PSET_IN_PEG_IN_VALUE = 0x08</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x08}</tt>
+** Value: The little endian 64-bit value of the peg-in for the Peg-in Transaction.
+*** <tt>{value}</tt>
+
+* Type: Peg-in Witness <tt>PSET_IN_PEG_IN_WITNESS = 0x09</tt>
+** Key: None. The key must only contain the 1 byte type.
+*** <tt>{0x09}</tt>
+** Value: The Peg-in witness for the Peg-in Transaction.
+*** <tt>{script}</tt>
```

Comment:
There should be `PSET_IN_ASSET_COMMITMENT` to supply asset commitment of the prevout, so Creator/Updater can provide blinding information for the inputs when they will not be the Blinder

Reply:
(this info is supplied in `auxiliary_generators` argument to `BlindTransaction()`

Reply:
This should be in the UTXO already.